spinnman/exceptions.py

```diff
@@ -16,17 +16,14 @@
 import traceback
 
 
-def get_physical_cpu_id(txrx, x, y, p):
-    if txrx is None:
-        return "Unknown Physical Core"
-    try:
-        cpu_info = txrx.get_cpu_information_from_core(x, y, p)
-        v_to_p_map = cpu_info.virtual_to_physical_core_map
-        if p >= len(v_to_p_map) or v_to_p_map[p] == 0xFF:
-            return "Unknown Physical Core"
-        return f"({v_to_p_map[p]})"
-    except Exception:
-        return "Unknown Physical Core"
+def get_physical_cpu_id(machine, sdp_header):
```

Comment:
I know it was already there but why is this method in exceptions and not in Machine?

Reply:
It could go in machine if it didn't look at the SDPHeader... as it is SpiNNMachine doesn't know what an SDPHeader is...